### PR TITLE
Ex 27 create contact and edit

### PR DESCRIPTION
**Pull request by sergiogaguero**

### CODE REVIEW

File: src/services/contacts.ts [hunk 1]
```diff
@@ -1,45 +1,86 @@
-// import axios from "axios";
-import { TEST_CONTACTS } from "../mock/mock";
+import axios from "axios";
+export const apiNearUrl = `${process.env.NEARAPI_BACKEND}`;'https://api.nearlogin.io';
```
Comment (leomanza) [inline src/services/contacts.ts:2]: was '**https://api.nearlogin.io'**;  added here on purpose?

**Review by leomanza — CHANGES_REQUESTED** [review]
just a comment about a hardcoded url

File: src/services/contacts.ts (hunk 1 above)
Comment (sergiogaguero) [reply src/services/contacts.ts:2]: my bad!, must be a copy/paste error.

Comment (sergiogaguero) [reply src/services/contacts.ts]: Fixed

**Pull request merged**